dbschema/src/main/java/org/folio/rest/persist/ddlgen/Index.java

```diff
@@ -83,24 +84,45 @@ public String getFinalSqlExpression(String tableLoc) {
           result .append(" , ");
         }
         String [] rawExpandedTerm = splitIndex[i].split("\\.");
-        StringBuilder expandedTerm = formatExpandedTerm(rawExpandedTerm);
-        result.append(tableLoc).append(".jsonb").append(expandedTerm);
+        StringBuilder expandedTerm = formatExpandedTerm(tableLoc + ".jsonb",rawExpandedTerm);
+        result.append(expandedTerm);
```

Comment:
Can you change these three lines to appendExpandedTerm(result, tableLoc, splitIndex[i])? This will move all expanding code into the method and improve readability.